src/app/app.component.ts

```diff
@@ -29,6 +30,10 @@ export class AppComponent implements OnInit, OnDestroy {
     public LoginForm: FormGroup;
     public modalRef;
     public title;
+    public item: any;
+    public regRegion: any;
+    public statisticGroup: any;
+    public clone: boolean;
```

Comment:
I don't think these are needed since these items aren't used elsewhere in the app component.